contracts/PvpArena.sol

```diff
@@ -381,6 +473,71 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         _lastActivityByCharacter[characterID] = block.timestamp;
     }
 
+    function _getCharacterPowerRoll(uint256 characterID, uint8 opponentTrait)
```

Comment:
As for now, we are not considering shields in this roll right?

Reply:
yup, there was a comment somewhere